SDNExpress/scripts/SDNExpressTenant.ps1

```diff
@@ -246,8 +246,15 @@ Configuration CreateTenantVMs
                     if ($($Using:node.productkey) -ne "" ) {
                         $key = "<ProductKey>$($Using:node.productkey)</ProductKey>"
                     }
-
-                    $finalUnattend = ($templateunattend -f $($Using:vminfo.ipaddress), $($Network.subnets[$using:vminfo.subnet].mask), $($Network.subnets[$using:vminfo.subnet].gateway), $($Network.DNSServers[0]), $($Using:vminfo.vmname), $($Using:Node.VMLocalAdminPassword), $key)
+		     if ($Using:node.UseIDns -eq $true) {
+                     write-verbose "Using iDNS"
+                     $finalUnattend = ($templateunattend -f "", "", "", "", $($Using:vminfo.vmname), $($Using:Node.VMLocalAdminPassword), $key, "true")

```

Comment:
Are you 100% that this works? I would have thought that by using "" instead of actual values as a replacement for some of the placeholder in the unattend, we would end up creating an invalid unattend file. For example, wouldn't this lead to an unattend file with the following (IP specified as "/" instead of a proper address + mask):

```
                    <UnicastIpAddresses>
                        <IpAddress wcm:action="add" wcm:keyValue="1">/</IpAddress>
                    </UnicastIpAddresses>
```

Unattend usually isn't very forgiving about invalid entries and it could lead to either the tenant VM not booting or the tenant VM showing the OOBE instead of booting straight to a finalized OS. Please confirm.